PopPUNK/models.py

```diff
@@ -561,7 +620,9 @@ def save(self):
              means=self.cluster_means,
              maxs=self.cluster_maxs,
              mins=self.cluster_mins,
-             scale=self.scale)
+             scale=self.scale,
+             assign_points = self.assign_points,
```

Comment:
Doesn't look like this is actually being saved?

Reply:
Sorry,  not sure why that would be here?

Reply:
Sorry the github diff made it look like this was in `save()` not `fit()`, my mistake